json.c

```diff
@@ -290,6 +290,29 @@ static struct json_object *path_info(int fd, uint32_t blob_id)
 	return obj;
 }
 
+static struct json_object *fb_info(int fd, uint32_t id)
+{
+	if (id == 0) {
+		return NULL;
+	}
+
+	// TODO: use drmModeGetFB2: https://patchwork.freedesktop.org/series/67552/
+	drmModeFB *fb = drmModeGetFB(fd, id);
```

Comment:
This seems to fail with EINVAL for me when used on the primary plane's FB, but the cursor one worked fine.
I haven't checked the kernel source, but maybe it doesn't like this when modifiers are in use, or something like that.

This call should check the return value.